mobile_verifier/src/coverage.rs

```diff
@@ -66,42 +65,45 @@ impl From<SignalLevelProto> for SignalLevel {
     }
 }
 
-pub struct CoverageDaemon<DT, GF> {
+pub struct CoverageDaemon<HA>
+where
+    HA: HexAssignment,
+{
     pool: Pool<Postgres>,
     auth_client: AuthorizationClient,
-    urbanization: Urbanization<DT, GF>,
+    hex_boost_data: HA,
     coverage_objs: Receiver<FileInfoStream<CoverageObjectIngestReport>>,
     initial_boosting_reports: Option<Vec<OracleBoostingReportV1>>,
     coverage_obj_sink: FileSinkClient,
     oracle_boosting_sink: FileSinkClient,
 }
 
-impl<DT, GF> CoverageDaemon<DT, GF>
+impl<HA> CoverageDaemon<HA>
 where
-    DT: DiskTreeLike,
-    GF: GeofenceValidator<u64>,
+    HA: HexAssignment,
 {
     pub async fn new(
         pool: PgPool,
         auth_client: AuthorizationClient,
-        urbanization: Urbanization<DT, GF>,
+        hex_boost_data: HA,
         coverage_objs: Receiver<FileInfoStream<CoverageObjectIngestReport>>,
         coverage_obj_sink: FileSinkClient,
         oracle_boosting_sink: FileSinkClient,
     ) -> anyhow::Result<Self> {
         tracing::info!("Setting initial values for the urbanization column");
 
-        let unassigned_hexes = UnassignedHex::fetch(&pool);
+        let unassigned_urbanization_hexes = UnassignedHex::fetch(&pool);
```

Comment:
This is more then just unassigned urbanization hexes, should be renamed to reflect that this is just unassigned hexes